projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -644,9 +648,11 @@ class ProjectorServer private constructor(
 
     if (hasWriteAccess) {
       PGraphicsEnvironment.clientDoesWindowManagement = toServerHandshakeEvent.clientDoesWindowManagement
-      PGraphicsEnvironment.setupDisplays(
-        toServerHandshakeEvent.displays.map { Rectangle(it.x, it.y, it.width, it.height) to it.scaleFactor })
-      with(toServerHandshakeEvent.displays[0]) { resize(width, height) }
+      SwingUtilities.invokeAndWait {
```

Comment:
Because it changes display configuration, updates Graphics on Windows, and fires a listener for that that eventually can run other code that requires EDT.

Reply:
Minor: so it fixes some bug, right? If you think it's worth mentioning in the changelog, please add it.